packages/webapp/public/locales/es/message.json

```diff
@@ -36,7 +36,8 @@
       "ADD": "Error: no se pudo agregar variedad a la base de datos",
       "ADD_ALREADY_EXISTS": "Error: existe variedad",
       "DELETE": "No se pudo eliminar la variedad de cultivo",
-      "UPDATE": "No se pudo actualizar la variedad de cultivo"
+      "UPDATE": "No se pudo actualizar la variedad de cultivo",
+      "FIND": "MISSING"
```

Comment:
Thank you for the translation @cfwm!

I think @dtrapplitefarm @kcussen usually have these done in a separate PR and in bulk with the other MISSING values but I am not certain.

Happy to add these in otherwise.